.github/CODEOWNERS

```diff
@@ -10,15 +10,10 @@
 /tests/modules/images                       @adamsilverstein @getsource
 /tests/testdata/modules/images              @adamsilverstein @getsource
 
-# Focus: JavaScript
-/modules/javascript                         @aristath @sgomes
-/tests/modules/javascript                   @aristath @sgomes
-/tests/testdata/modules/javascript          @aristath @sgomes
-
-# Focus: Site Health
-/modules/site-health
-/tests/modules/site-health
-/tests/testdata/modules/site-health
+# Focus: CSS & JS
+/modules/css-and-js                         @aristath @sgomes
+/tests/modules/css-and-js                   @aristath @sgomes
+/tests/testdata/modules/css-and-js          @aristath @sgomes
```

Comment:
Per the latest clarification, this should be "JS & CSS" (`js-and-css`) throughout.

Reply:
Focus area updated throughout.